src/main/java/com/emt/service/MatchService.java

```diff
@@ -1,41 +1,61 @@
 package com.emt.service;
 
+import com.emt.entity.Player;
 import com.emt.mapper.MatchMapper;
 import com.emt.model.exception.IdenticalPlayersException;
 import com.emt.model.request.CreateMatchRequest;
 import com.emt.model.response.MatchResponse;
 import com.emt.repository.MatchRepository;
+
 import java.util.List;
 import java.util.Optional;
+
 import lombok.RequiredArgsConstructor;
 import org.springframework.stereotype.Service;
 
 @Service
 @RequiredArgsConstructor
 public class MatchService {
 
-  private final MatchRepository matchRepository;
-  private final MatchMapper matchMapper;
+    private final MatchRepository matchRepository;
+
+    private final MatchMapper matchMapper;
+
+    private final PlayerService playerService;
+
+    private static final int CONSTANT_K = 30;
 
-  private final PlayerService playerService;
+    public List<MatchResponse> getAllMatches() {
+        return matchRepository.findAll().stream().map(matchMapper::mapToResponse).toList();
+    }
+
+    public MatchResponse createMatch(CreateMatchRequest request) {
```

Comment:
Create more UnitTests that will assert elo rating changes (you can use parametrized tests for this)